blocks/library/gallery/gallery-image.js

```diff
@@ -1,8 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
```

Comment:
Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```